ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-release-4.10__okd.yaml

```diff
@@ -0,0 +1,64 @@
+base_images:
+  base:
+    name: "4.10"
+    namespace: origin
+    tag: base
+  ocp_builder_rhel-8-golang-1.16-openshift-4.9:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.16-openshift-4.9
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.16-openshift-4.9:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9
```

Comment:
any reason not to use rhel-8-golang-1.16-openshift-4.10?

Reply:
yes, that's the one currently still being used for OCP on master: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml#L33 (lmk if you'd like me to update that too)

Reply:
Actually, the still unused config for release-4.10 has it set correctly: https://github.com/openshift/release/blob/master/ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-release-4.10.yaml#L33. Should I change just update release-4.10__okd, or also master__okd?

Reply:
Ok, weird. I was sure I saw 4.9 on the first link above, but apparently I was wrong. So, nvmd, I'll update this to 4.10, too.